README.md

```diff
@@ -1,34 +1,58 @@
 
-This repo contains examples of PyPRT usage. You can find the source code in the main [pyprt](https://github.com/Esri/pyprt) repository. More detailed information is available there.
+This repo contains examples of PyPRT usage. You can find installation instructions and the source code in the main [PyPRT](https://github.com/Esri/pyprt) repository.
 
-<p align="center">
-  <img src="images/building.png">
-</p>
-
-<p align="center">
-  <img src="images/generated_trees.png">
-</p>
+<p align="center"><img src="images/building.png" alt="green building" /></p>
+<p align="center"><img src="images/generated_trees.png" alt="city with trees" /></p>
 
 ## Table of Contents
 
-* [Installation](#installation)
-* [Examples](#examples)
-* [Running the examples](#running-the-examples-using-pipenv)
+* [Requirements](#requirements)
+* [Running the examples](#running-the-examples)
+* [Available Examples](#available-examples)
 * [Provided Rule Packages](#provided-rule-packages)
 * [License](#license)
 
+## Requirements
+
+* Windows 10 64bit or Linux 64bit (RHEL7/CentOS7 compatible distro)
+* Python 3.6 64bit or later. Please note, we currently provide PyPRT builds for Python 3.6 and Python 3.8. For other Python versions, please [build](https://github.com/Esri/pyprt) PyPRT yourself. 
+* Packages (see `envs` directory): pyprt, arcgis, notebook, vispy, PyQT5, scipy, PyGEL3D, tornado
+
+## Running the examples
 
-## Installation
+PyPRT can be installed with `pip install pyprt` or `conda install -c esri pyprt`. To run the examples a few more packages are required, so please read on! :-)
 
-Simply run `pip install pyprt` in your desired Python environment or `conda install -c esri pyprt` in a Conda environment. Then use `import pyprt` in your scripts.
+### Using virtualenv and pip
 
+1. Open a shell and change to the root of this example repository.
+1. First time setup:
+   1. Create a virtualenv: `python3.8 -m venv .venv` (replace `python3.8` with path to desired Python version if necessary)
+   1. Update to latest `pip`:
+      * Linux: `.venv/bin/python -m pip install --upgrade pip`
+      * Windows: `.venv\Scripts\python -m pip install --upgrade pip`
+   1. Install required packages for the example - this will fetch PyPRT from PyPI and all packages necessary to run the examples (replace `py38` with the used Python version):
+      * Linux: `.venv/bin/python -m pip install -r envs/py38/requirements.txt`
+      * Windows: `.venv\Scripts\python -m pip install -r envs\py38\requirements.txt`
+1. Activate the environment:
+   * Linux: `source .venv/bin/activate`
+   * Windows: `.venv\Scripts\activate.bat`
+1. Now run the examples, e.g. `python ex1_python_encoder.py` 
+1. For the examples based on Jupyter Notebooks, first start jupyter with `jupyter notebooks` and then open the desired example notebook in the opened browser window.
```

Comment:
The command is `jupyter notebook` (without the s at the end).

Reply:
fixed